modules/rls/rls.c

```diff
@@ -281,7 +282,6 @@ static int mod_init(void)
                 return -1;
         }
         rls_integrated_xcap_server = xcap_api.integrated_server;
-        db_url = xcap_api.db_url;
```

Comment:
A bit confused, it seems like this already exists by using the value of 
`modparam("xcap", "db_url", "some_db_url_here:3306")` wouldn't it be easier to just set that to the local one?